sweep/txgenerator.go

```diff
@@ -56,7 +56,7 @@ func generateInputPartitionings(sweepableInputs []input.Input,
 	// on the signature length, which is not known yet at this point.
 	yields := make(map[wire.OutPoint]int64)
 	for _, input := range sweepableInputs {
-		size, _, err := getInputWitnessSizeUpperBound(input)
+		size, _, err := input.WitnessType().GetWitnessSizeUpperBound()
```

Comment:
The input doesn't know the witness size, it would only delegate the call to the witness type.

But I was able to eliminate the `Custom` type by removing the `Type()` method from the `WitnessType`. The numeric type is only used in `utxonursery` and `breacharbiter` and they work fine by only using the standard witness types.